modules/overrides/config.go

```diff
@@ -114,12 +114,16 @@ type LocalBlocksOverrides struct {
 	CompleteBlockTimeout time.Duration `yaml:"complete_block_timeout,omitempty" json:"complete_block_timeout,omitempty"`
 }
 
+type HostInfoOverrides struct {
+	HostIdentifiers []string `yaml:"host_identifiers,omitempty" json:"host_identifies,omitempty"`
+	MetricName      string   `yaml:"metric_name,omitempty" json:"metric_name,omitempty"`
```

Comment:
does this need to be per tenant configurable? user overrides are already quite complex and it would be better to only put in them what is necessary

Reply:
it would be ideal if we could configure this per tenant re: @cedricziel's comment on the processor PR here: https://github.com/grafana/tempo/pull/4698#discussion_r1955904886